src/widgets/histogram/chart.js

```diff
@@ -940,6 +950,46 @@ module.exports = cdb.core.View.extend({
     return axis;
   },
 
+  _setupFillColor: function () {
+    if (!this._widgetModel.isAutoStyleEnabled()) {
+      return false;
+    }
+
+    var obj = this._widgetModel.getAutoStyle();
+    var data = this.model.get('data');
+
+    if (!_.isEmpty(obj)) {
+      var geometryDefinition = obj.definition[Object.keys(obj.definition)[0]]; // Gets first definition by geometry
+      var colorsRange = geometryDefinition && geometryDefinition.color.range;
+      var domain = [(_.first(data).min || _.first(data).start), (_.last(data).max || _.last(data).end)];
+      var autoStyleColorsScale;
+
+      if (domain && colorsRange) {
+        autoStyleColorsScale = d3.scale.quantize()
+          .domain(domain)
+          .range(colorsRange);
+      }
+
+      this._autoStyleColorsScale = autoStyleColorsScale;
+    }
+  },
+
+  _getFillColor: function (d, i) {
```

Comment:
Just being picky here but... can we use more descriptive argument names than `d`?

Argument `i` doesn't seem to be used in the function, as well.